src/ontology/components/hra_subset.owl

```diff
@@ -1,14 +1,17 @@
 <?xml version="1.0"?>
-<rdf:RDF xmlns="http://purl.obolibrary.org/obo/uberon/components/hra_subset.owl#"
-     xml:base="http://purl.obolibrary.org/obo/uberon/components/hra_subset.owl"
+<rdf:RDF xmlns="http://purl.obolibrary.org/obo/uberon/component-download-hra_subset.owl#"
+     xml:base="http://purl.obolibrary.org/obo/uberon/component-download-hra_subset.owl"
      xmlns:obo="http://purl.obolibrary.org/obo/"
```

Comment:
Is this change really expected, @matentzn?

Reply:
Nothing dramatic. This change is probably ok.

Reply:
Can you make a PR on Makefile.jinja two that replaces cp $< $@ with ROBOT annotate -i $< .... -o $@?

For now, so that protege can "see" the import you can overwrite the goal in uberon.Makefile

Reply:
Fixe is in the PR https://github.com/INCATools/ontology-development-kit/pull/931